v03_pipeline/lib/tasks/delete_project_table.py

```diff
@@ -0,0 +1,18 @@
+import luigi
+
+from v03_pipeline.lib.paths import project_table_path
+from v03_pipeline.lib.tasks.base.base_delete_table import BaseDeleteTableTask
+from v03_pipeline.lib.tasks.files import GCSorLocalTarget
+
+
+class DeleteProjectTableTask(BaseDeleteTableTask):
```

Comment:
Should `DeleteProjectFamilyTablesTask` be a dependency of this task?

Reply:
oof, excellent question.  my initial thought was no, that we should be able to delete a project able in isolation.  however, I'm re-thinking that.

Reply:
I'm gonna go ahead and add the dependency.  It's not clear to me what the right move is... but might as well put it there for completeness!